terraform/modules/gro_ingestion_service/lambdas/test/functions/dbItem.ts

```diff
@@ -0,0 +1,53 @@
+export const dbItem = {
```

Comment:
Should we add this to a mocks or consts folder?

Reply:
yes